models/user.rb

```diff
@@ -61,12 +61,14 @@ def to_hash(params={})
           ).count
 
           # Note that the comments may have been responses to a thread not started by author_id.
-          comment_thread_ids = Comment.where(
+          comment_thread_ids = Comment.includes(:comment_thread).where(
```

Comment:
This is a little nitpicky, so feel free to ignore, but a comment as to why the `includes(:comment_thread)` is needed might be nice. It took me a minute to figure out why the associated threads needed to be loaded from Mongo.